roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -392,6 +392,7 @@
       size: "{{ (openshift_logging_elasticsearch_pvc_size | trim | length == 0) | ternary('10Gi', openshift_logging_elasticsearch_pvc_size) }}"
       access_modes: "{{ openshift_logging_elasticsearch_pvc_access_modes | list }}"
       pv_selector: "{{ openshift_logging_elasticsearch_pvc_pv_selector }}"
+      storage_class_name: "{{ openshift_logging_elasticsearch_pvc_storage_class_name | default('', true) }}"
```

Comment:
In order for the PVC to dynamically use the cloud provider configured to generate an underlying PV, the storage class spec needs to be omitted.
What use case are you trying to resolve with this patch?

Reply:
@ewolinetz well I have 2 storage classes, "standard", the default one on HDD and "ssd", and additional one on SSD. Without this patch, ansible creates a pv using the standard storageclass, even though I specified "ssd"

Reply:
@mcanevet ah, I see. Currently we would recommend that you don't set `openshift_logging_elasticsearch_pvc_dynamic=True` in your inventory, however that may not be the correct change.

With this change, if you do not specify a storage class name with `openshift_logging_elasticsearch_pvc_storage_class_name` would it use your default storage class?

Reply:
@mcanevet I'm not following you on that first part.. why could you not set a different cluster size if you don't use dynamic?
Ack to the second part

Reply:
Since this task is identical to the one above it (the only difference is the when condition), could you instead remove the `when` statements and delete one of the tasks? We likely should just deprecate `openshift_logging_elasticsearch_pvc_dynamic`